### PR TITLE
Create a retry and dlq strategy only fo Topic and Direct exchanges

### PR DESCRIPTION
**Pull request by andrelugomes**
@juliofalbo 

Trying to solve #11 issue.

### CODE REVIEW

File: spring-rabbitmq-tuning-lib/src/main/java/com/tradeshift/amqp/autoconfigure/TunedRabbitAutoConfiguration.java [hunk 1]
```diff
@@ -293,6 +296,18 @@ private void autoCreateQueues(TunedRabbitProperties properties, RabbitAdmin rabb
         }
     }
 
+    private boolean isAValidExchange(TunedRabbitProperties properties) {
+        return DIRECT.equals(properties.getExchangeType()) || TOPIC.equals(properties.getExchangeType());
```
Comment (juliofalbo) [inline spring-rabbitmq-tuning-lib/src/main/java/com/tradeshift/amqp/autoconfigure/TunedRabbitAutoConfiguration.java]: To follow your pattern (that is cool) you can create a method called `isATopicExchange` and change here for something like this:
```
private boolean isAValidExchange(TunedRabbitProperties properties) {
        return isADirectExchange(properties) || isATopicExchange(properties));
}
```

BTW, nice job and thank you very much for your contribution!

Comment (andrelugomes) [reply spring-rabbitmq-tuning-lib/src/main/java/com/tradeshift/amqp/autoconfigure/TunedRabbitAutoConfiguration.java]: good idea

Comment (andrelugomes) [reply spring-rabbitmq-tuning-lib/src/main/java/com/tradeshift/amqp/autoconfigure/TunedRabbitAutoConfiguration.java]: done....

**Comment by juliofalbo** [comment]
Hey man!
Can you rebase your branch with master and squash your commits? After this I can merge!

**Pull request closed without merging**